_dev/apps/ui/src/store/modules/product-feed/state.ts

```diff
@@ -34,15 +34,25 @@ export type ProductFeedReport = {
 }
 export interface ProductFeedSettings {
   shippingSetup: ShippingSetupOption|null;
-  estimateCarriers: CustomCarrier[];
-  shippingSettings: ShopShippingInterface[];
-  rate: RateType|null;
-  deliveryDetails: DeliveryDetail[];
+  estimateCarriers?: CustomCarrier[];
+  shippingSettings?: ShopShippingInterface[];
+  rate?: RateType|null;
```

Comment:
What's the reason behind this change?

Reply:
that was because of how things were done in `createProductFeedApiPayload`